rcsb/exdb/chemref/PubChemIndexCacheProvider.py

```diff
@@ -320,7 +322,7 @@ def updateMissing(self, expireDays=0, fetchLimit=None, updateUnmatched=True, num
         try:
             # ---
             # Get current the indices of source chemical reference data -
-            ok, ccidxP, ccsidxP = self.__rebuildChemCompSourceIndices(numProc, **kwargs)
+            ok, ccidxP, ccsidxP = self.__rebuildChemCompSourceIndices(numProcChem, **kwargs)
```

Comment:
@piehld To answer your questions above. I renamed this to  `numProcChem` so that `numProc` applies to the PubChem requests. This Chem Comp stuff already was defaulting to 12, so that is not a change I made. Interestingly, I noticed that in `MultiProcUtil`, the default for `numProc` is `multiprocessing.cpu_count() * 2`.

Since obviously the rename of the variable is a bit confusing, I can rename to `numProcChemComp `and add a bit more information to the comments to clarify.

Reply:
OK, thanks for clarifying. So we can leave `numProc` defaulted to 1, but I still feel like we should reduce `numProcChemComp` to 8. I'm wondering if John had it at 12 because he was always planning to run it from his hp server...

Reply:
I ended up updating `numProc` to 2 (I already made the change before seeing your comment) and `numProcChemComp` to 8. I think 2 will be unlikely to hit the request limits. Let me know if you think it should go back to 1 for some reason.

Reply:
That sounds good to me!